packages/core/__tests__/fakers/batch-faker.ts

```diff
@@ -0,0 +1,33 @@
+import { fakerPT_BR as faker } from '@faker-js/faker'
+import { Batch } from '../../src/domain/entities'
+import { BatchDto } from '../../src/dtos'
+
+export class BatchesFaker {
+  static fake(baseDto?: Partial<BatchDto>, productsIds?: string[]) {
+    return Batch.create(BatchesFaker.fakeDto(baseDto, productsIds))
+  }
+  static fakeDto(baseDto?: Partial<BatchDto>, productsIds?: string[]): BatchDto {
```

Comment:
I think that it would be better using a `for` loop to create the batchs of a product and getting the `productId` from `BatchDto` to make the bind between the entities.